codex-cli/tests/config.test.tsx

```diff
@@ -1,6 +1,11 @@
 import type * as fsType from "fs";
 
-import { loadConfig, saveConfig } from "../src/utils/config.js"; // parent import first
+import {
+  loadConfig,
+  saveConfig,
+  DEFAULT_SHELL_MAX_BYTES,
+  DEFAULT_SHELL_MAX_LINES,
+} from "../src/utils/config.js"; // parent import first
```

Comment:
you can remove the comment while you're here. imports should be auto-sorted, so I'm not sure why this is here